### ISSUE TITLE
[nodejs][11.qnamaker] Sample crash if QnA service name does not match .bot file

### ISSUE THREAD

**Issue by enzocanoo** [Microsoft/BotBuilder-Samples#618]
## Sample information
1. Sample type: **samples**
2. Sample language: **nodejs**
3. Sample name: 11.qnamaker

## Describe the bug
The QnA service name in [`index.js`](https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/javascript_nodejs/11.qnamaker/index.js#L39) does not match the name in the [`bot.recipe`](https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/javascript_nodejs/11.qnamaker/deploymentScripts/msbotClone/bot.recipe#L13) provided.
- `index.js`
  ```C#
  const QNA_CONFIGURATION = 'qnamakerService';
  ```
- `bot.recipe`
  ```json
  {
    "type": "qna",
    "id": "34",
    "name": "qnamaker-QnAMaker"
  },
  ```

Also, the `bot.recipe` does not create all resources because needs the `34.qna` spec.

## To Reproduce
1. Follow the instructions to run the bot.
4. The bot crash when trying to resolve the QnA service keys.

## Expected behavior
The sample should work out-to-the-box or provides instructions to modify the `QNA_CONFIGURATION` constant.

## Additional context
The file `34.qna` could be copied from the [TypeScript sample](https://github.com/Microsoft/BotBuilder-Samples/blob/master/samples/javascript_typescript/11.qnamaker/deploymentScripts/msbotClone/34.qna).

[bug]

**Comment by JasonSowers** [comment]
@enzocanoo was this fixed in one of your PRs?

**Comment by enzocanoo** [comment]
Hi @JasonSowers,

Not this one. However, taking into account that the `.bot` files of the samples were deleted, this sample needs to be updated to include instructions to generate the file.

**Comment by JasonSowers** [comment]
CLosing as this will be addressed in the new readmes #828

**Comment by cbtham** [comment]
This is still not fixed
